operate/qa/integration-tests/src/test/java/io/camunda/operate/webapp/security/sso/AuthenticationWithPersistentSessionsIT.java

```diff
@@ -125,6 +125,7 @@
     },
     properties = {
       "server.servlet.context-path=" + AuthenticationWithPersistentSessionsIT.CONTEXT_PATH,
+      "spring.web.resources.add-mappings=true",
```

Comment:
❓ What does this do?

Reply:
It allows fetching static resources (like index.html, javascript sources, ...). If not enabled, no mappings are (magically) created and cannot be served by Spring.

That setting is necessary because the current `application.properties` (in the `dist` module) disables resource mappings, causing these test cases to fail. Previously, it was not necessary doing this because Spring enabled it by default and the `application.properties` was not on the classpath of the integration tests.